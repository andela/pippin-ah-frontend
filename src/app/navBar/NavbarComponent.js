import React from 'react'

const NavbarComponent = () => {
    return (
        <nav className="navbarbg">
            <div className="nav-wrapper">
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
                <i className="far fa-play-circle" /> &nbsp;
                <a href="index.html" className="brand-logo">
                    LearnGround
                </a>
                <ul id="nav-mobile" className="right hide-on-med-and-down">
                    <li>
                        <a href="Science.html">Science</a>
                    </li>
                    <li>
                        <a href="Technology.html">Technology</a>
                    </li>
                    <li>
                        <a href="Technology.html">Engineering</a>
                    </li>
                    <li>
                        <a href="Arts.html">Arts</a>
                    </li>
                    <li>
                        <a href="Mathematics.html">Mathematics</a>
                    </li>

                    <li>
                        <a href="SignIn.html">SignIn</a>
                    </li>
                    <li>|</li>

                    <li>
                        <a href="SignUp.html">SignUp</a>
                    </li>
                </ul>
            </div>
        </nav>
    )
}
export default NavbarComponent
