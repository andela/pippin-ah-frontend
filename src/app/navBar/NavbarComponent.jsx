import React from 'react'

const NavbarComponent = () => {
    return (
        <nav className="navbarbg">
            <div className="nav-wrapper" id="iconRight">
                <div className="Logo">
                    <i className="far fa-play-circle" />
                    <a href="#!" className="brand-logo">
                        LearnGround
                    </a>
                </div>
                <div>
                    <a
                        href="#!"
                        data-target="mobile-demo"
                        className="sidenav-trigger"
                    >
                        <i className="material-icons">menu</i>
                    </a>
                    <ul className="right hide-on-med-and-down">
                        <li>
                            <a href="#!">Science</a>
                        </li>
                        <li>
                            <a href="#!">Technology</a>
                        </li>
                        <li>
                            <a href="#!">Engineering</a>
                        </li>
                        <li>
                            <a href="#!">Arts</a>
                        </li>
                        <li>
                            <a href="#!">Mathematics</a>
                        </li>
                        <li>
                            <a href="#!">SignIn</a>
                        </li>
                        <li>|</li>

                        <li>
                            <a href="#!">SignUp</a>
                        </li>
                    </ul>
                </div>
            </div>

            <ul className="sidenav" id="mobile-demo">
                <li>
                    <a href="#!">Science</a>
                </li>
                <li>
                    <a href="#!">Technology</a>
                </li>
                <li>
                    <a href="#!">Engineering</a>
                </li>
                <li>
                    <a href="#!">Arts</a>
                </li>
                <li>
                    <a href="#!">Mathematics</a>
                </li>
                <li>
                    <a href="#!">SignIn</a>
                </li>
                <li>|</li>

                <li>
                    <a href="#!">SignUp</a>
                </li>
            </ul>
        </nav>
    )
}
export default NavbarComponent
