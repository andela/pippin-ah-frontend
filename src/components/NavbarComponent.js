import React from 'react';


const NavbarComponent  = () => {

  return (
    <nav className="navbarbg">
    <div className="nav-wrapper">
      <a href="" className="brand-logo">LearnGround</a>
      <ul id="nav-mobile" className="right hide-on-med-and-down">
        <li><a href="Science.html">Science</a></li>
        <li><a href="Technology.html">Technology</a></li>
        <li><a href="Technology.html">Engineering</a></li>
        <li><a href="Arts.html">Arts</a></li>
        <li><a href="Mathematics.html">Mathematics</a></li>
      </ul>
    </div>
  </nav>
        
  );
  
}
export default NavbarComponent; 
