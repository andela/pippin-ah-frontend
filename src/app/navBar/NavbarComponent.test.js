import React from 'react'
import { shallow } from 'enzyme'
import NavbarComponent from './NavbarComponent'

describe(' Component', () => {
    it('should render the NavBar', () => {
        const component = shallow(<NavbarComponent />)
        expect(component.exists()).toBe(true)
        expect(component).toMatchSnapshot()
    })
})
