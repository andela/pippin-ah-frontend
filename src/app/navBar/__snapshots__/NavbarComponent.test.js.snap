// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Component should render the NavBar 1`] = `
<nav
  className="navbarbg"
>
  <div>
    <div
      className="nav-wrapper"
      id="iconRight"
    >
      <div
        className="Logo"
      >
        <i
          className="far fa-play-circle"
        />
            
        <a
          className="brand-logo"
          href="/"
        >
          LearnGround
        </a>
      </div>
      <div
        className="categories"
      >
        <ul
          className="right hide-on-med-and-down"
        >
          <li>
            <Link
              replace={false}
              to="/subject"
            >
              Science
            </Link>
          </li>
          <li>
            <Link
              replace={false}
              to="/subject"
            >
              Technology
            </Link>
          </li>
          <li>
            <Link
              replace={false}
              to="/subject"
            >
              Engineering
            </Link>
          </li>
          <li>
            <Link
              replace={false}
              to="/subject"
            >
              Arts
            </Link>
          </li>
          <li>
            <Link
              replace={false}
              to="/subject"
            >
              Mathematics
            </Link>
          </li>
        </ul>
        <a
          className="sidenav-trigger"
          data-target="mobile-demo"
          href="#!"
        >
          <i
            className="material-icons"
          >
            menu
          </i>
        </a>
      </div>
      <div
        className="hide-on-med-and-down show"
      >
        <ul>
          <li />
          <li>
            <Link
              replace={false}
              to="/login"
            >
              SignIn
            </Link>
          </li>
          <li>
            |
          </li>
          <li>
            <Link
              replace={false}
              to="/signup"
            >
              SignUp
            </Link>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <ul
    className="sidenav"
    id="mobile-demo"
  >
    <li>
      <a
        href="#!"
      >
        Science
      </a>
    </li>
    <li>
      <a
        href="#!"
      >
        Technology
      </a>
    </li>
    <li>
      <a
        href="#!"
      >
        Engineering
      </a>
    </li>
    <li>
      <a
        href="#!"
      >
        Arts
      </a>
    </li>
    <li>
      <a
        href="#!"
      >
        Mathematics
      </a>
    </li>
    <li>
      <a
        href="#!"
      >
        SignIn
      </a>
    </li>
    <li>
      |
    </li>
    <li>
      <a
        href="#!"
      >
        SignUp
      </a>
    </li>
  </ul>
</nav>
`;
